doc/UserGuide.md

```diff
@@ -44,10 +44,10 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > be seen using the `viewall` command.
 > 
 > Persons can have any number of tags (including 0)
-
+> Address must be in the following format: `BLOCK, STREET, UNIT, POSTALCODE`
```

Comment:
:+1:  for mentioning the format and giving appropriate examples in the user guide.